docs/pages/webhook/events/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Events'
+---
+
+# List of events
+
+| Event               | Description                                    | Detail                                        |  Available  |
+| ------------------- | ---------------------------------------------- | --------------------------------------------- | :---------: |
+| AUCTION_BID_CREATED | Occurs whenever a bid on an auction is created | [detail](/webhook/events/AUCTION_BID_CREATED) |     ✅      |
```

Comment:
```suggestion
| Event               | Triggered                                    | Detail                                        |  Available  |
| ------------------- | ---------------------------------------------- | --------------------------------------------- | :---------: |
| AUCTION_BID_CREATED | when a bid on an auction is created | [detail](/webhook/events/AUCTION_BID_CREATED) |     ✅      |
```
or :
```suggestion
| Event               | Triggered when                                    | Detail                                        |  Available  |
| ------------------- | ---------------------------------------------- | --------------------------------------------- | :---------: |
| AUCTION_BID_CREATED | a bid on an auction is created | [detail](/webhook/events/AUCTION_BID_CREATED) |     ✅      |
```

Reply:
I kept a mix of `Triggered` and `whenever ...` this way, it can be read in both directions like `AUCTION_BID_CREATED` whenever ...